### PR TITLE
[AKS] fix get-credentials without directory

### LINKED ISSUE #4849: Add tests around `az aks get-credentials`
#4762 was merged to fix a Windows-specific bug, but could use additional test coverage.

### PR DESCRIPTION
**Pull request by mboersma**
Closes #4849.
Closes Azure/AKS#42.

### CODE REVIEW

File: src/command_modules/azure-cli-acs/HISTORY.rst [hunk 1]
```diff
@@ -2,11 +2,15 @@
...
 Release History
 ===============
+
+2.0.21
+++++++
+* fix `aks get-credentials -f` without fully-qualified path
+
 2.0.20
 ++++++
 * `acs create`: emit out an actionable error if provisioning application failed for lack of permissions
```
Comment (williexu) [inline src/command_modules/azure-cli-acs/HISTORY.rst]: Put your new history under 2.0.20 and pre-check should pass.
Master is still on 2.0.19

**Comment by williexu** [comment]
Tests failing :-/

**Comment by mboersma** [comment]
Tests pass for me though, with `run_tests --module acs`. It's just in the two `PURPOSE="Automation"` jobs where it errors out.

How can I reproduce the failure in Travis CI locally?
```
$ ./scripts/ci/test_automation.sh 
Missing version string
```

**Comment by williexu** [comment]
`run_tests --module acs` should also run the two tests that are failing. I'm surprised that all of the tests pass for you. 
You can run the specific tests that are failing with:
`run_tests --module acs --test test_aks_commands.AzureKubernetesServiceScenarioTest.test_aks_create_with_upgrade`
(replace the ending with the other failing test to run the other)

I just tried running the `test_aks_create_with_upgrade` test myself with your recording and I get the same error as in Travis CI.
I also ran the test with `--live` and I get a `CloudError: 400 Client Error: Bad Request for url:`

**Comment by williexu** [comment]
@mboersma can you try deleting all the recordings in `acs` (should be four of them), re-recording all the tests by running them live, and then playing back all the tests using those recordings?

**Comment by mboersma** [comment]
@williexu I would love to, but AKS is heavily oversubscribed and the chances of me getting a clean `az acs` + `az aks` test run are very low currently. I have been trying to regenerate the recordings but no luck so far. I'll keep trying.

**Comment by mboersma** [comment]
@williexu I noticed our tests could contend over a local file where the ACS service principal is cached. So I rewrote the `az aks create` commands to use an explicit SP and secret instead, then regenerated the VCR yaml on a clean run of `run_tests --module acs`.

**Comment by mboersma** [comment]
@williexu this should be ready to merge now.

**Pull request merged**